### PR TITLE
Implemented Showing Hitbox during Slow Mode

### PR DESCRIPTION
**Pull request by Alko-li**
Added hitbox image whenever slow mode is implemented

### CODE REVIEW

File: Assets/Game/Character/Rhea/Rhea.cs [hunk 1]
```diff
@@ -9,6 +9,8 @@ public class Rhea : Character<Rhea, RheaState, RheaStateInput>
     private float dashDuration = 0.2f;
     [SerializeField]
     private GameObject shieldPrefab = null;
+    [SerializeField]
+    private GameObject hitbox;
```
Comment (reverienest) [inline Assets/Game/Character/Rhea/Rhea.cs:13]: variable names should alternate cases: EX - hitBox

File: Assets/Game/Character/Scripts/SharedCharacterController.cs [hunk 2]
```diff
@@ -17,26 +17,37 @@ public class SharedCharacterController : MonoBehaviour
 
     [SerializeField]
     private GameObject circlePrefab = null;
+    [SerializeField]
+    private GameObject hitbox;
```
Comment (reverienest) [inline Assets/Game/Character/Scripts/SharedCharacterController.cs:21]: Same statement with variable cases here

File: Assets/Game/Character/Rhea/Rhea.cs (hunk 1 above)
Comment (BrandonShockley) [reply Assets/Game/Character/Rhea/Rhea.cs:13]: Around the internet I see both "hit box" (hitBox in camel case) and "hitbox" (hitbox in camel case) being used. I used the latter in the ticket description, so I think it's fine to leave it as is.

**Review by BrandonShockley — COMMENTED** [review]
Gotta fix these merge conflicts

**Comment by BrandonShockley** [comment]
Sorry, Albert, but I went in and implemented it this morning. The ticket is done. I appreciate the effort you made though.

**Pull request closed without merging**